awx/main/access.py

```diff
@@ -317,8 +317,10 @@ def check_license(self, add_host_name=None, feature=None, check_expiration=True)
             validation_info['time_remaining'] = 99999999
             validation_info['grace_period_remaining'] = 99999999
 
-        report_violation = lambda message: logger.error(message)
-
+        if quiet:
+            report_violation = lambda message: None
+        else:
+            report_violation = lambda message: logger.warning(message)
```

Comment:
@wenottingham is your ask that this *stays* at `logger.error` ?

Reply:
No, see above edited comment